### DISCUSSION TITLE
i18n support for content (products, collections...)

### DISCUSSION THREAD

**Discussion by mariusa** [vendurehq/vendure#789] [General]
Is there i18n support for content (products, collections...) ? Similar to https://apps.shopify.com/langshop

I don't see a Languages section in online admin demo.

**Comment by michaelbromley** [comment]
Yes, i18n is supported! You first need to specify which languages you want to support - this is done in the GlobalSettings page:
![image](https://user-images.githubusercontent.com/6275952/112296043-cdc1a300-8c94-11eb-9601-faa5fb9b045f.png)

Once set up, you'll see a language switcher when you view products, collections etc. detail pages.

**Reply by mariusa** [reply]
Great, thanks for clarification!
